SU2_CFD/src/output/CAdjFlowCompOutput.cpp

```diff
@@ -450,7 +450,7 @@ void CAdjFlowCompOutput::LoadSurfaceData(CConfig *config, CGeometry *geometry, C
 bool CAdjFlowCompOutput::SetInit_Residuals(CConfig *config){
   
   return (config->GetTime_Marching() != STEADY && (curInnerIter == 0))|| 
-        (config->GetTime_Marching() == STEADY && (curTimeIter < 2)); 
+        (config->GetTime_Marching() == STEADY && (curInnerIter < 2)); 
```

Comment:
(not your job but) can you add some () around the == STEADY to be consistent with other places please (some compilers warn about this)

Reply:
done!